### ISSUE TITLE
Test the build with fable-compiler

### ISSUE THREAD

**Issue by et1975** [fable-elmish/elmish#105]
Since packaging doesn't build with fable-compiler anymore have to add some test project that does.

**Comment by alfonsogarciacaro** [comment]
Adding a test project that references the main project and checks some functions [as in Fable.PowerPack](https://github.com/fable-compiler/fable-powerpack/tree/master/tests) is probably a good option :)

> Note in the case of Fable.PowerPack the actual test command is in a package.json in the repo root, but as you don't need package.json anymore for Fable libs you can also put in the tests folder.

**Comment by jr-jesse123** [comment]
Hello, is this still relevant? I can try it with some instructions.

**Comment by et1975** [comment]
Yes, it would be nice to have JS-hosted tests to complement the CLR-hosted
ones. I think Alfonso's original suggestion is obsolete at this point, so
I'd look at the current fable-compiler codebase for inspiration on how to
setup the test/tooling.

>
>
